src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyListener);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyListener);
+  }
+
+  keyListener = (event: KeyboardEvent) => {
+    this.setState({ key: event.key });
+  };
+
+  render(): React.ReactNode {
+    return (
+      <div className="App">
+        {
+          this.state.key
+            ? <p className="App__message">{`The last pressed key is [${this.state.key}]`}</p>
+            : <p className="App__message">Nothing was pressed yet</p>
+        }
```

Comment:
in both cases it's a similar paragraphes, so better to select strings by ternary operator

Reply:
and better to destructure state